source/priority-queue.ts

```diff
@@ -32,6 +32,27 @@ export default class PriorityQueue implements Queue<RunFunction, PriorityQueueOp
 		this.#queue.splice(index, 0, element);
 	}
 
+	setPriority(id: string, priority?: number) {
+		const existingIndex: number = this.#queue.findIndex((element: Readonly<PriorityQueueOptions>) => element.id === id);
+		const [item] = this.#queue.splice(existingIndex, 1);
```

Comment:
Please add tests for when an invalid id is passed.

I think this code has a bug where the `-1` returned by `Array#findIndex` causes the last item to be spliced, instead of no item at all.